.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: python
 python:
   - 2.7
+  - 3.6
```

Comment:
```suggestion
```

Given https://github.com/CottageLabs/negotiator/pull/2 hasn't been released yet, we can drop this line and atleast get the code changes in which work on both Pythons